--- ideviceinstaller-1.1.1/src/ideviceinstaller.c.orig	2020-06-07 17:02:44.000000000 +0200
+++ ideviceinstaller-1.1.1/src/ideviceinstaller.c	2023-11-27 21:08:55.034355773 +0100
@@ -299,7 +299,7 @@ static int zip_get_contents(struct zip *
 static int zip_get_app_directory(struct zip* zf, char** path)
 {
 	int i = 0;
-	int c = zip_get_num_files(zf);
+	int c = zip_get_num_entries(zf, 0);
 	int len = 0;
 	const char* name = NULL;
 
